crmsh/bootstrap.py

```diff
@@ -1134,22 +1133,59 @@ def init_ssh():
     Configure passwordless SSH.
     """
     utils.start_service("sshd.service", enable=True)
-    configure_local_ssh_key()
+    for user in ["root", "hacluster"]:
```

Comment:
This `["root", "hacluster"]` list looks something that should go in a constant, as it is repeated later

Reply:
Changed